test/unit_tests/braket/experimental/autoqasm/test_parameters.py

```diff
@@ -729,15 +783,16 @@ def parametric(alpha: float, beta: float):
 
     expected = """OPENQASM 3.0;
 input float[64] alpha;
+float[64] beta = 1.5;
 qubit[1] __qubits__;
 bool __bool_0__;
-__bool_0__ = alpha && 1.5;
+__bool_0__ = alpha && beta;
 if (__bool_0__) {
     rx(alpha) __qubits__[0];
 }
 bit __bit_1__;
 __bit_1__ = measure __qubits__[0];"""
-    assert parametric(FreeParameter("alpha"), 1.5).to_ir() == expected
+    assert parametric.make_bound_program({"beta": 1.5}).to_ir() == expected
```

Comment:
Weird, that out of all the outputs only this one changed. I don't think it's anything to worry about for this PR though

Reply:
It's because beta isn't a free parameter, it's inlined during program creation. I will update this test like all the others